src/components/description_list/description_list_title.tsx

```diff
@@ -10,14 +10,23 @@ import React, { HTMLAttributes, FunctionComponent } from 'react';
 import classNames from 'classnames';
 import { CommonProps } from '../common';
 
+type EuiDescriptionListType = 'row' | 'inline' | 'column' | 'responsiveColumn';
+
+export interface EuiDescriptionListTitleProps extends CommonProps {
+  type?: EuiDescriptionListType;
```

Comment:
Let's try this:

```suggestion
  /**
   * Adds a line-break opportunity `<wbr>` element after the `<dt>`
   */
  enforceLineBreaks?: boolean;
```

And set it to `false` by default.